### PR TITLE
Fix fetch request in ListBusinesses component

### PR DESCRIPTION
**Pull request by Pidgett**

### CODE REVIEW

File: src/components/ListBusinesses.js [hunk 1]
```diff
@@ -41,10 +42,10 @@ const ListBusinesses = () => {
       </tr> */}
           {business.map((listing) => (
             <tr>
-              <td>{listing.name}</td>
+              <td>{listing.bname}</td>
```
Comment (m3ia) [inline src/components/ListBusinesses.js:45]: was changing from "name" to "bname" intentional? for "business name"?

File: src/components/ListBusinesses.js [hunk 2]
```diff
@@ -41,10 +42,10 @@ const ListBusinesses = () => {
...
               <td>{listing.city}</td>
               <td>{listing.country}</td>
-              <td>{listing.lift}</td>
+              <td>{listing.lift ? "✅" : "❎"}</td>
```
Comment (m3ia) [inline src/components/ListBusinesses.js:48]: I like these! Just a tip: for emojis and symbols, I got complaints from Prettier. If you happen to run into the same issues, you can find the UTF-8 equivalents for these symbols [here](https://www.w3schools.com/charsets/ref_utf_dingbats.asp)! If you don't run into issues, nvm! 

WHITE HEAVY CHECK MARK
https://www.w3schools.com/charsets/tryit.asp?deci=9989
NEGATIVE SQUARED CROSS MARK
https://www.w3schools.com/charsets/tryit.asp?deci=10062

File: src/index.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import { Auth0Provider } from "@auth0/auth0-react";
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById("root")
```
Comment (m3ia) [inline src/index.js:12]: Ahh I see Prettier might be making a ton of complaints haha. Just a tip-- if you run into Prettier issues and hover your mouse over, click "Quick Fix" and have it automatically fix all the Prettier errors in the file.

File: server/index.js [hunk 4]
```diff
@@ -50,18 +50,27 @@ app.get("/businesses/:bType", async (req, res) => {
...
-        const { name, bType, city, state, zipCode, country, lift, liftType, elevator, valet, publicTransit } = req.body;
-        const updateBusiness = await pool.query(
-            "UPDATE businesses SET"
-        )
-    } catch (error) { 
-        console.error(error.message);
-    }
-})
-*/
+app.put("/todos/:id", async (req, res) => {
```
Comment (m3ia) [inline server/index.js:53]: This is interesting. Does this work? I set mine up differently. If it works--no worries! If not, let me know and we can pair to make this work with George's set-up.

**Review by m3ia — COMMENTED** [review]
Not a ton of changes so if you need to merge to main to see what's going on on Heroku, feel free to merge! I think all is good to go except I'm just curious about the PUT request in the server/index.js file. If it works--looks good to me!

**Review by yjwangit — APPROVED** [review]
Nice progress on the project! Auth0 has been implemented; various components work!

**Pull request merged**